muxers/yamux/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.20.0 [2020-06-30]
+
+- Updated dependencies.
```

Comment:
Maybe we should make the CHANGELOG format more consistent. Sometimes a single change such as here is made a bullet point, sometimes it is not. I would only use bullet points when listing multiple changes, but would not mind if we decide to always use them, as long as we can have it consistent.

Reply:
Ok.

Reply:
See https://github.com/libp2p/rust-libp2p/pull/1640/commits/49c964fc591cb24f0f76ae51c9393355a489a4c2.